tests/clar_libgit2.c

```diff
@@ -558,5 +558,36 @@ bool cl_sandbox_supports_8dot3(void)
 
 	return supported;
 }
+
+bool cl_sandbox_supports_reparse_points(void)
+{
+	unsigned int flags = 0;
+	char path[8];
+	char drive = 'A' + _getdrive() - 1;
+
+	sprintf(path, "%c:\\", drive);
+
+	GetVolumeInformation(path, NULL, 0, NULL, NULL, &flags, NULL, 0);
```

Comment:
This isn't, I suppose, super critical, but you can mount filesystems within the space of another filesystem.  So strictly speaking, I think that we should query the mount point for the clar sandbox path (`GetVolumePathName`?) and then pass *that* to `GetVolumeInformation`.  Otherwise you could mount a VHD at (say) `C:\Temp\Foo`, and use that as your clar path and we would incorrectly query the volume information for `C:\`.

I would suggest that this is not particularly useful *except* that this PR is about making our ability to run tests in wacky places (ramdisks, in your specific example) more useful.